betago/model.py

```diff
@@ -57,7 +65,7 @@ def next_move():
             json_result = jsonify(**result)
             return json_result
 
-        self.app.run(host='0.0.0.0', debug=True, threaded=True, use_reloader=False)
+        self.app.run(host='0.0.0.0', port=int("8081"), debug=True, threaded=True, use_reloader=False)
```

Comment:
I suggest using `argparse` to read the port from the command line, see https://github.com/maxpumperla/betago/blob/be14775b9e19bfdde41caacb2620935a4637cdf8/examples/train_generator.py#L15 for an example.

Then the user will start the demo with `python run_demo.py --port=8081` to specify what port they want to use. That will work in any environment